assets/css/telia_ace.css

```diff
@@ -1,3 +1,16 @@
+#telia-ace-leijuke {
+  bottom: 20px;
+  color: #ffffff;
+  position: fixed;
+  right: 20px;
+  z-index: 1;
+}
+.telia-chat-leijuke {
+  min-width: 212px;
```

Comment:
Tried to match the width of the button coming from Telia ACE, but since the placeholder and loading text is variable it's actually kind of useless. This has been removed.